static/js/pivotal.js

```diff
@@ -197,7 +253,7 @@
             for (pt_id in pivotal_ids) {
               var id = pivotal_ids[pt_id];
 
-              getPivotalInfo(PIVOTAL_TOKEN, id, function(info) {
+              getPivotalStoryInfo(PIVOTAL_TOKEN, id, function(info) {
                 display_message[id] = info;
                 injectPivotalStatus(project, display_message);
```

Comment:
Not sure if this is important in this case, but this method will append a new child to `$project`, which will do a repaint/reflow (I think).
If you experience relatively slow UX when clicking the button, you can rework that part to store the children in an array and add them all at once.